layout-management-webapps/src/main/webapp/vue-app/site-navigation/components/SiteNavigationDrawer.vue

```diff
@@ -130,6 +146,10 @@ export default {
         })
         .finally(() => this.loading = false);
     },
+    retrieveSites(){
```

Comment:
try instead to return the site root node directly in the getNavigationNodes service response